RuntimeLicenses/license.txt

```diff
@@ -9,7 +9,8 @@ except to the extent those have additional terms.
 IF YOU COMPLY WITH THESE LICENSE TERMS, YOU HAVE THE RIGHTS BELOW.
 
 1. INSTALLATION AND USE RIGHTS. 
-You may install and use any number of copies of the software to develop and test your applications. 
+You may only use the C# Extension for Visual Studio Code with Visual Studio Code, Visual Studio or Xamarin Studio software to
+help you develop and test your applications.
```

Comment:
@andschwa let me start with the disclaimer that I am not a lawyer. But to answer your question, this extension has two licenses -- there is a license of the source code itself, which is MIT. There is also a license on the official download of the extension that Microsoft produces and uploads to the extension gallery. That is what this is. It is NOT MIT.